lib/helper/logging.helper.js

```diff
@@ -0,0 +1,11 @@
+const winston = require('winston');
+
+const logger = winston.createLogger();
```

Comment:
In a future version I'm going to remove the Winston dependency since it doesn't really use their formatting advantages.
Making it able to log to somewhere else might also be interesting. However for now logging is only used in debug modus.